src/main/kotlin/org/jetbrains/research/transformations/Transformation.kt

```diff
@@ -15,12 +15,12 @@ interface Transformation {
      * @param [toStoreMetadata] - flag that indicates necessity
      * of metadata storing.
      */
-    fun apply(tree: ITree, toStoreMetadata: Boolean)
+    fun apply(tree: TreeContext, toStoreMetadata: Boolean)
```

Comment:
Sorry, I am not sure, but maybe it will be better to rename the "tree" parameter into "treeCtx" or smth like that since we changed the type to TreeContext?

Reply:
Sure, you are right, thanks!